### ISSUE TITLE
Markdown: New list item (* or -) not appearing automatically on new line of a list

### ISSUE THREAD

**Issue by balaziks** [Automattic/simplenote-electron#371]
When editing a Markdown list in a Android app the editor adds new list item character (\* or -) automatically when hitting enter at the end of a list item.

This doesn't happen on the Windows desktop app.

**Comment by drw158** [comment]
This is expected — we have not implemented this feature yet.

**Comment by nfcampos** [comment]
(this should be much easier after #370)

**Comment by dmsnell** [comment]
I'm closing this issue @balaziks because it's a duplicate of #314. We will track development there, but thanks for the report!

**Comment by dmsnell** [comment]
Not a duplicate. Sorry!

**Comment by drw158** [comment]
I think this has been implemented.
